src/table.rs

```diff
@@ -301,74 +213,66 @@ impl ToString for Row {
 }
 
 impl Table {
-    pub fn create_sentence(&self) -> String {
-        let mut sentence = String::new();
-        sentence.push_str("CREATE TABLE ");
-        sentence.push_str(&self.name);
-        sentence.push_str(" (");
+    pub fn create_statement(&self) -> String {
+        let mut statement = String::new();
+        statement.push_str("CREATE TABLE ");
+        statement.push_str(&self.name);
+        statement.push_str(" (");
         for (i, col) in self.cols.iter().enumerate() {
             if i > 0 {
-                sentence.push_str(", ");
+                statement.push_str(", ");
             }
-            sentence.push_str(&col.name);
-            sentence.push(' ');
+            statement.push_str(&col.name);
+            statement.push(' ');
             match &col.column_type {
-                ColumnType::Int => sentence.push_str("INT"),
+                ColumnType::Int => statement.push_str("INT"),
                 ColumnType::String(collation) => {
-                    sentence.push_str("VARCHAR(10)");
+                    statement.push_str("VARCHAR(10)");
                     if let Some(collation) = collation {
-                        sentence.push_str(" COLLATE ");
-                        sentence.push_str(collation);
+                        statement.push_str(" COLLATE ");
+                        statement.push_str(collation);
```

Comment:
I think the statement construction can be simplified by making using of [`join`](https://doc.rust-lang.org/std/primitive.slice.html#method.join) to connect parts with commas

Reply:
Makes sense. It was written by Github Copilot :)